app/models/role.rb

```diff
@@ -128,7 +128,7 @@ def add_permissions(permissions, options = {})
     search = options.delete(:search)
 
     collection = Permission.where(:name => permissions).all
-    raise ArgumentError, 'some permissions were not found' if collection.size != permissions.size
+    raise ArgumentError, 'some permissions were not found' if collection.size < permissions.size
```

Comment:
Why are you adding permissions that already exist in another plugin?  I don't understand from the issue description (empty) how you're triggering this.